code/modules/mob/living/carbon/superior_animal/robots/subtype/mining/mining_drone.dm

```diff
@@ -117,14 +172,36 @@
 	M.GetDrilled() // Mine the turf
 	return TRUE
 
+/mob/living/carbon/superior_animal/robot/mining/verb/force_mine()
+	set name = "Mine"
+	set desc = "Mine a close by rock."
+	set category = "Mob verbs"
+
+	mine()
+
 /mob/living/carbon/superior_animal/robot/mining/proc/pick_ore(var/obj/item/ore/O)
 	visible_message("[src] pick up [O]") // Visible message
 	O.forceMove(src) // Pick up the item
 	return TRUE
 
+/mob/living/carbon/superior_animal/robot/mining/verb/force_pick_ore()
+	set name = "Pick up ore"
+	set desc = "Pick up some close by ore."
+	set category = "Mob verbs"
+
+	pick_ore()
+
 /mob/living/carbon/superior_animal/robot/mining/proc/drop_loot()
 	visible_message("[src] empties \his storage bin.") // Visible message
 	var/turf/T = get_turf(src)
 	for(var/obj/O in contents) // Empty everything
 		O.forceMove(T) // Move the item on the ground
 	return TRUE
+
+/mob/living/carbon/superior_animal/robot/mining/verb/force_drop_loot()
```

Comment:
Also can you check if those verbs won't be able to be manually activated by non-controlling ghosts?

Reply:
they HAVE to be the mob to get these verbs